widgets/ScrollTable.lua

```diff
@@ -229,7 +229,6 @@ local methods = {
 				end
 
 				cell:SetHeight(rowHeight);
-				cell:SetWidth(self.columns[j].width);
```

Comment:
Are you sure this line needs to be removed?

Reply:
I will investigate and get back to you. I removed it for some reason I can't remember off the top of my head why.

Reply:
Okay so it looks like it's not really necessary to be there but it doesn't hurt anything anyways. Since the OnSizeChanged handler resizes all the column widths anyways, setting the individual cell widths in each column is not necessary.